prometheus_client/twisted.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+from . import REGISTRY, generate_latest, CONTENT_TYPE_LATEST
+
+from twisted.web.resource import Resource
+
+
+class PrometheusMetrics(Resource):
```

Comment:
Should `Resource` be in the name to indicate more of what this is? `PrometheusMetrics` is a bit on the vague side.